### PR TITLE
doc: new post on webapp dev

### PR DESCRIPTION
**Pull request by jsilvela**
put a webapp inside kubernetes with CloudNativePG

### CODE REVIEW

**Comment by sxd** [comment]
@jsilvela the links are pointing to localhost :)

File: content/blog/developing-webapps-with-cloudnative-pg/index.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+```
+
+You should be able to see the image `myapp` in the `docker images` listing.
+Now we'd like to *load* this image into our KinD cluster's nodes.
+
+``` sh
+kind load docker-image myapp:latest --name webapp-demo
+```
+
+(*) You could have uploaded your dockerfile into a public container registry,
```
Comment (sxd) [inline content/blog/developing-webapps-with-cloudnative-pg/index.md:139]: Dockerfile

File: content/blog/developing-webapps-with-cloudnative-pg/index.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+`pq: the database system is shutting down`.
+Recovery should be no longer than 1-2 seconds, as a replica would be
+promoted by the CloudNativePG operator.
+Since the webapp was written against the service, it will recover once
+the new primary is in place.
+
+But you might think that for read-only endpoints like the page showing
+the latest stock values, the webapp could leverage the `cluster-example-ro`
+service.
+If if did, there would not be an outage when we killed the primary.
```
Comment (sxd) [inline content/blog/developing-webapps-with-cloudnative-pg/index.md:401]: If if*

**Comment by jsilvela** [comment]
Thanks for looking @sxd !! I've fixed some. Three of them do need to be `localhost`...

**Comment by jsilvela** [comment]
Any comment on the content @sxd ? Too long, not long enough? More adventure?

**Comment by sxd** [comment]
> Any comment on the content @sxd ? Too long, not long enough? More adventure?

I think that you can include something related to pgbouncer and also escalate the web app and show the connections internally, will be useful

**Review by gbartolini — DISMISSED** [review]

**Review by leonardoce — APPROVED** [review]

**Pull request merged**